README.md

```diff
@@ -619,6 +620,44 @@ Example with `tasty-discover`:
 ```
 This returns an error in HLS if 'tasty-discover' is not in the path: `could not execute: tasty-discover`.
 
+## Troubleshooting
+
+### Common issues
+
+#### Difficulties with Stack and `Paths_` modules
+
+These are known to be somewhat buggy at the moment: https://github.com/haskell/haskell-language-server/issues/478.
```

Comment:
Stack 2.5 is officially released now, so this is "kind of" a solved thing? Still good to have it here for a while maybe..

Reply:
Agree, with a note pointing out stack versions newer than 2.5 fix it